core/src/packet_deserializer_stage.rs

```diff
@@ -96,10 +174,7 @@ impl PacketDeserializer {
                 }
             }
 
-            if start.elapsed() >= recv_timeout
-                || packet_count_overflowed
-                || packets_received >= packet_count_upperbound
```

Comment:
Not sure if would hit corner cases by removing this condition (eg `packet_count_upperbound`). Potential scenario:  if banking_stage buffer only has one free spot (eg `packet_count_upperbound == 1`), and `packet_batch_receiver` has tons of packets. `receive_until()` would keep receiving packets and putting them into unbounded channel until `timeout`. Would that increase memory footprint, or is this not an issue?

Reply:
It could potentially be an issue, but I don't think that potential new with this change. Previously they'd have queued up in the receiver from sigverify_stage, now they'd just be queued up in the receiver from deserializer_stage.